packages/material-ui/src/Tab/Tab.js

```diff
@@ -45,18 +45,15 @@ const TabRoot = experimentalStyled(ButtonBase, {
 })(({ theme, styleProps }) => ({
   /* Styles applied to the root element. */
   ...theme.typography.button,
-  maxWidth: 264,
-  minWidth: 72,
+  maxWidth: 360,
+  minWidth: 90,
   position: 'relative',
   minHeight: 48,
   flexShrink: 0,
-  padding: '6px 12px',
+  padding: '12px 16px',
```

Comment:
this value is according to the spec.

Reply:
It seems. to break the top icon case:

<img width="490" alt="Capture d’écran 2021-05-26 à 12 49 27" src="https://user-images.githubusercontent.com/3165635/119647863-1296c100-be21-11eb-9d40-38ece0df09b8.png">


https://material.io/components/tabs#anatomy

<img width="424" alt="Capture d’écran 2021-05-26 à 12 49 19" src="https://user-images.githubusercontent.com/3165635/119647861-11fe2a80-be21-11eb-935c-9b551fc3edb9.png">

https://deploy-preview-26458--material-ui.netlify.app/components/tabs/#icon-tabs

How about?

```diff
diff --git a/packages/material-ui/src/Tab/Tab.js b/packages/material-ui/src/Tab/Tab.js
index bab7db3ef9..08fcbfdc94 100644
--- a/packages/material-ui/src/Tab/Tab.js
+++ b/packages/material-ui/src/Tab/Tab.js
@@ -62,6 +62,7 @@ const TabRoot = experimentalStyled(ButtonBase, {
     styleProps.label && {
       minHeight: 72,
       paddingTop: 9,
+      paddingBottom: 9,
       [`& .${tabClasses.wrapper} > *:first-child`]: {
         marginBottom: 6,
       },
```